src/diffusers/pipelines/unclip/pipeline_unclip_image_variation.py

```diff
@@ -0,0 +1,454 @@
+# Copyright 2022 Kakao Brain and The HuggingFace Team. All rights reserved.
```

Comment:
Is this so? Or is it just Hugging Face for the code? Just wondering, no idea how those things work!

Reply:
This is the licensing @patrickvonplaten added to the text to image pipeline. We should probably clarify with him :)

Reply:
Think it's fine to mention Kakao Brain, since we use their code as a reference when implementing it here.